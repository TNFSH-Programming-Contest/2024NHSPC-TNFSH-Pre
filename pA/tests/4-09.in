734620857675318396 989739761240688789 553828242942513647
428457976928648753 996363873303839828 853367011626032251
