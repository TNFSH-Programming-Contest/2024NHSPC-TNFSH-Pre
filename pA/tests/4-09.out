1598160923595766941
